### PR TITLE
preference loss sign is inverted and leads to negative loss

### PR DESCRIPTION
**Pull request by winglian**
## Summary
In testing cases where there is no alpha/NLL loss used, the loss becomes negative, which is probably not the intended behavior. see https://github.com/huggingface/trl/blob/main/trl/trainer/dpo_trainer.py#L1234

- Hardware Type: <BLANK>
- [ ] run `make test` to ensure correctness
- [ ] run `make checkstyle` to ensure code style
- [ ] run `make test-convergence` to ensure convergence

### CODE REVIEW

**Review by austin362667 — APPROVED** [review]
Great catch. Thank you

**Review by austin362667 — CHANGES_REQUESTED** [review]
Would you mind updating the corresponding tests? They failed in my CI as expected.

**Comment by austin362667** [comment]
I believe the issue lies here: https://github.com/linkedin/Liger-Kernel/blob/0bb6c72d03f0fc570b9e954c42b723ab7e0c315f/test/utils.py#L514.

**Review by austin362667 — APPROVED** [review]
Thank you!

File: src/liger_kernel/chunked_loss/fused_linear_preference.py [hunk 1]
```diff
@@ -408,7 +408,7 @@ def _compute_loss(
         else:
             preference_loss, aux_outputs = preference_loss_outputs, []
 
-        loss = alpha * chosen_nll_loss - preference_loss
```
Comment (shivam15s) [inline src/liger_kernel/chunked_loss/fused_linear_preference.py:411]: Seems like this is correct for all losses except dpo_loss. I think the change the sign change should be here instead: https://github.com/linkedin/Liger-Kernel/blob/21bacccd107b40146e599e2c27af46d7d157f174/src/liger_kernel/chunked_loss/dpo_loss.py#L52

**Review by shivam15s — CHANGES_REQUESTED** [review]

File: src/liger_kernel/chunked_loss/fused_linear_preference.py (hunk 1 above)
Comment (austin362667) [reply src/liger_kernel/chunked_loss/fused_linear_preference.py:411]: Hi @shivam15s  Thanks for noticing this. What are your thoughts on negating each preference loss term to align with the formulas in the docstrings? This would allow us to maintain the base preference structure as `nll_loss + preference_loss` while making both terms represent losses to be minimized.

Comment (shivam15s) [reply src/liger_kernel/chunked_loss/fused_linear_preference.py:411]: Sure @austin362667 ! I think that might help with readability.
If you have some time, could you add the fix to this PR?

Comment (austin362667) [reply src/liger_kernel/chunked_loss/fused_linear_preference.py:411]: Ya sure! I might need to open a new PR (base on this one) cc @winglian
